scripts/cleanup_failed_prs.sh

```
@@ -0,0 +1,145 @@
+
+# Get all open PRs
+echo -e "${BLUE}📋 Fetching open PRs...${NC}"
+prs=$(gh pr list --state open --json number,title,headRefName,author,labels,updatedAt --limit 50)
+
+# Parse and categorize PRs
+failed_test_prs=()
+formatting_prs=()
+dependency_prs=()
+main_prs=()
+
+echo "$prs" | jq -r '.[] | [.number, .title, .headRefName, .author.login] | @tsv' | while IFS=$'\t' read -r number title branch author; do
```

Shellcheck (0.10.0)

[warning] branch appears unused. Verify use (or export if used externally).

(SC2034)